src/config/config.ts

```diff
@@ -780,23 +780,27 @@ export class Config implements IConfig {
       }
 
       const handleAlias = (alias: string, hidden = false) => {
-        if (this._commands.has(alias)) {
+        const aliasWithDefaultTopicSeparator = alias.replaceAll(' ', ':')
```

Comment:
We have a util function for whenever we need to do this:

```ts
import { toStandardizedId } from '../util/ids'
const aliasWithDefaultTopicSeparator = toStandardizedId(alias)
```

Reply:
🤦 I should have noticed this, my bad